src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -99,12 +100,58 @@ protected String name() {
             return "Property Mutator";
         }
 
-        private final Caller mMutator;
-        private final Caller mAccessor;
+        private final PropertyCaller mMutator;
+        private final PropertyCaller mAccessor;
         private final WeakHashMap<View, Object> mOriginalValues;
         private final Object[] mOriginalValueHolder;
     }
 
+    public static class LayoutSetVisitor extends ViewVisitor {
+        public LayoutSetVisitor(List<Pathfinder.PathElement> path, LayoutCaller mutator) {
+            super(path);
+            mOriginalValues = new WeakHashMap<View, int[]>();
+            mMutator = mutator;
```

Comment:
While the LayoutCaller is technically a mutator, we don't have to use such general naming here- we know it's a layout, and it'll never be anything else.